apps/srt-file-transmit.cpp

```diff
@@ -105,6 +106,10 @@ int main( int argc, char** argv )
     if (sf == "" || !false_names.count(sf))
         ::g_skip_flushing = true;
 
+    string sfull = Option<OutString>(params, "no", o_fullstats);
+    if (sfull == "" || !false_names.count(sfull))
+        ::total_stats = true;
```

Comment:
This seems to be a variable used in transmitmedia, so please keep `transmit_` prefix. In future we'd like to make some rework around it, so this is to keep clear about what it is.

Reply:
> This seems to be a variable used in transmitmedia, so please keep transmit_ prefix. In future we'd like to make some rework around it, so this is to keep clear about what it is.

OK, I'll do it!

Reply:
Done! :smile: